bin/stashcp2/tests/test.sh

```diff
@@ -0,0 +1,4 @@
+python stashcp.py --debug user/taburaad/public/2gb_file.tar .
```

Comment:
You should be able to copy most of the HTCondor-CE's travis.yml.  And probably copy all of the setup_tests.sh file.